webapi/src/main/java/org/sakaiproject/webapi/controllers/SitesController.java

```diff
@@ -1,26 +1,32 @@
 /******************************************************************************
- * Copyright 2015 sakaiproject.org Licensed under the Educational
- * Community License, Version 2.0 (the "License"); you may not use this file
- * except in compliance with the License. You may obtain a copy of the License at
- *
- * http://opensource.org/licenses/ECL-2.0
- *
- *  Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- ******************************************************************************/
+* Copyright 2015 sakaiproject.org Licensed under the Educational
```

Comment:
We can use the new template here:

https://github.com/sakaiproject/sakai/blob/master/ECLv2-header-template.txt